AstSemantics.md

```diff
@@ -82,38 +82,47 @@ only communicate through messaging, e.g. in browsers using `postMessage`. It
 will be possible to share linear memory between threads of execution when
 [threads](PostMVP.md#threads) are added.
 
-### Linear Memory Operations
-
-Linear memory operations are annotated with a memory type and perform a
-conversion between that memory type and a local type.
-
-Loads read data from linear memory, convert from their memory type to a basic
-type, and return the result:
-
-  * `i32.load8_s`: sign-extend i8 to i32
-  * `i32.load8_u`: zero-extend i8 to i32
-  * `i32.load16_s`: sign-extend i16 to i32
-  * `i32.load16_u`: zero-extend i16 to i32
-  * `i32.load`: (no conversion)
-  * `i64.load`: (no conversion)
-  * `f32.load`: (no conversion)
-  * `f64.load`: (no conversion)
-
-Stores have an operand providing a value to store. They convert from the value's
-local type to their memory type, and write the resulting value to linear memory:
-
-  * `i32.store8`: wrap i32 to i8
-  * `i32.store16`: wrap i32 to i16
-  * `i32.store`: (no conversion)
-  * `i64.store`: (no conversion)
-  * `f32.store`: (no conversion)
-  * `f64.store`: (no conversion)
-
-Wrapping of integers simply discards any upper bits; i.e. wrapping does not
-perform saturation, trap on overflow, etc.
-
-In addition to storing a value to linear memory, store instructions also
-reproduce their value operand, with no conversion applied.
+### Linear Memory Accesses
+
+Linear memory access is accomplished with explicit `load` and `store` operations that
+accept as input an `index` operand that is interpreted as a unsigned byte offset into the
+linear memory.
```

Comment:
They accept as input an `index` and an `offset` which are described in the [Addressing](https://github.com/WebAssembly/design/blob/master/AstSemantics.md#addressing) section immediately following this section.